.github/workflows/ci.yml

```diff
@@ -89,6 +89,9 @@ jobs:
       - name: Build project
         run: sbt '++ ${{ matrix.scala }}' test
 
+      - name: Test sbt plugin
+        run: sbt 'zioHttpGenSbt/scripted'
```

Comment:
@987Nabil this was naive, not sure how to add sbt plugin testing to CI.
this does not work:
```
[error] java.lang.RuntimeException: ci.yml does not contain contents that would have been generated by sbt-github-actions; try running githubWorkflowGenerate
[error] 	at scala.sys.package$.error(package.scala:30)
[error] 	at sbtghactions.GenerativePlugin$.reportMismatch$1(GenerativePlugin.scala:919)
[error] 	at sbtghactions.GenerativePlugin$.compare$1(GenerativePlugin.scala:925)
[error] 	at sbtghactions.GenerativePlugin$.$anonfun$projectSettings$5(GenerativePlugin.scala:929)
[error] 	at sbtghactions.GenerativePlugin$.$anonfun$projectSettings$5$adapted(GenerativePlugin.scala:906)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[error] 	at java.base/java.lang.Thread.run(Thread.java:1583)
[error] (githubWorkflowCheck) ci.yml does not contain contents that would have been generated by sbt-github-actions; try running githubWorkflowGenerate
```

How can I add a CI step to run `sbt zioHttpGenSbt/scripted`?